### ISSUE TITLE
Support WiFi for R77

### ISSUE THREAD

**Issue by thrust26** [stella-emu/stella#817]
It would be nice if the R77 would get net access via an USB WiFi adapter. E.g. to support PlusROMs.

Thoughts:
- WiFi on Linux is nothing simple
- R77 uses an old Linux version which doesn't support many modern chipsets
- however some adapters with supported chipsets are still available (starting at ~$5)
- we probably have to limit the number of supported chipsets (maybe only one). 
- access via configuration file only
- connection status display in Stella

**Comment by sa666666** [comment]
Since the R77 port is effectively dead post 7.0, do we even want to keep this here now?

**Comment by thrust26** [comment]
@DirtyHairy  I think you wanted to continue. Is that still valid?

**Comment by sa666666** [comment]
Actually R77 support might be dead as of 7.0; I don't remember.  But it's certainly dead for 8.0, as the code has been removed, and we've moved to SDL3 (R77 barely supported SDL2).  I have no interest in working on that console, as the whole affair frankly left a bad taste in my mouth.  But @DirtyHairy, if you want to work on the 6.x version, feel free.  But I'd really like your help and expertise with the current codebase 🥺

**Comment by DirtyHairy** [comment]
I agree; the R77 is basically dead and, realistically, I will not work on this issue.
